### PR TITLE
chore: upgrade deps

### PR DESCRIPTION
**Pull request by sbsrnt**
Upgrades:
https://github.com/viktorgullmark/exilence-next/issues/575
https://github.com/viktorgullmark/exilence-next/issues/576
https://github.com/viktorgullmark/exilence-next/issues/577

Fixes:
https://github.com/viktorgullmark/exilence-next/issues/579

### CODE REVIEW

File: ExilenceNextApp/src/components/item-table/ItemTableContainer.tsx [hunk 1]
```diff
@@ -184,7 +184,7 @@ const ItemTableContainer = ({
               anchorEl={anchorEl}
             />
             {hideableColumns.length > 1 && (
-              <Tooltip title={t('label.toggle_visible_columns')} placement="bottom">
+              <Tooltip title={t('label.toggle_visible_columns') || ''} placement="bottom">
```
Comment (viktorgullmark) [inline ExilenceNextApp/src/components/item-table/ItemTableContainer.tsx:187]: Whats the point of this? We should have the translations loaded for this right?

File: ExilenceNextApp/src/components/item-table/ItemTableContainer.tsx [hunk 2]
```diff
@@ -194,7 +194,7 @@ const ItemTableContainer = ({
                 </IconButton>
               </Tooltip>
             )}
-            <Tooltip title={t('label.toggle_stash_tab_filter')} placement="bottom">
+            <Tooltip title={t('label.toggle_stash_tab_filter') || ''} placement="bottom">
```
Comment (viktorgullmark) [inline ExilenceNextApp/src/components/item-table/ItemTableContainer.tsx:197]: See above comment

File: ExilenceNextApp/src/components/item-table/ItemTableContainer.tsx [hunk 3]
```diff
@@ -205,7 +205,7 @@ const ItemTableContainer = ({
                 <FilterListIcon />
               </IconButton>
             </Tooltip>
-            <Tooltip title={t('label.toggle_export_menu')} placement="bottom">
+            <Tooltip title={t('label.toggle_export_menu') || ''} placement="bottom">
```
Comment (viktorgullmark) [inline ExilenceNextApp/src/components/item-table/ItemTableContainer.tsx:208]: See above comment

File: ExilenceNextApp/src/index.tsx [hunk 4]
```diff
@@ -49,7 +49,7 @@ enableLogging({
 
 configure({ enforceActions: 'observed' });
 
-moment.locale(electronService.remote.app.getLocale());
+moment.locale(electronService.app.getLocale());
```
Comment (viktorgullmark) [inline ExilenceNextApp/src/index.tsx:52]: This seems to crash locally

File: ExilenceNextApp/src/utils/snapshot.utils.ts [hunk 5]
```diff
@@ -51,10 +51,8 @@ export const mapSnapshotsToStashTabPricedItems = (snapshot: Snapshot, stashTabs:
     });
 };
 
-export const getSnapshotCardValue = (snapshotCount: number) => {
-  let label = `${snapshotCount}${snapshotCount >= 1000 ? '+' : ''}`;
-  return label;
-};
+export const getSnapshotCardValue = (snapshotCount: number) =>
+  `${snapshotCount}${snapshotCount >= 1000 ? '+' : ''}`;
```
Comment (viktorgullmark) [inline ExilenceNextApp/src/utils/snapshot.utils.ts:55]: Dont we need a return here or am I mistaken?

File: ExilenceNextApp/src/utils/snapshot.utils.ts [hunk 6]
```diff
@@ -105,29 +99,19 @@ export const formatValue = (
...
-  return snapshots
-    .map((s) => {
-      const values: number[] = [
-        moment(new Date(s.created).getTime()).valueOf(),
-        +getValueForSnapshot(s).toFixed(2),
-      ];
-      return values;
-    })
+export const formatSnapshotsForChart = (snapshots: IApiSnapshot[]): number[][] =>
+  snapshots
```
Comment (viktorgullmark) [inline ExilenceNextApp/src/utils/snapshot.utils.ts:103]: See above comment

File: ExilenceNextApp/tsconfig.json [hunk 7]
```diff
@@ -18,7 +18,10 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react"
+    "jsx": "react",
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": false,
```
Comment (viktorgullmark) [inline ExilenceNextApp/tsconfig.json:23]: I dislike the noImplicitAny, but I guess we need it atm :D

Comment (sbsrnt) [reply ExilenceNextApp/tsconfig.json:23]: Yeah for the time being I went with `noImplicitAny` due to `react-table` as it seemed to keep biting itself with DOM attributes type-wise 🤔

File: ExilenceNextApp/src/index.tsx (hunk 4 above)
Comment (sbsrnt) [reply ExilenceNextApp/src/index.tsx:52]: Yeah, the `remote` is depracated so need some alternative to that 🤔

File: ExilenceNextApp/src/utils/snapshot.utils.ts (hunk 5 above)
Comment (sbsrnt) [reply ExilenceNextApp/src/utils/snapshot.utils.ts:55]: Technically `() => { return 'foo' }` is the same as `() => 'foo'`.

File: ExilenceNextApp/src/components/item-table/ItemTableContainer.tsx (hunk 2 above)
Comment (sbsrnt) [reply ExilenceNextApp/src/components/item-table/ItemTableContainer.tsx:197]: No clue why `TS` was yelling about insufficient types from `Tooltip`. It looked like `t()` is undefined on initialization hence the errors but that shouldn't be true to how we load the translations 🤔

File: ExilenceNextApp/src/components/item-table/ItemTableContainer.tsx (hunk 1 above)
Comment (sbsrnt) [reply ExilenceNextApp/src/components/item-table/ItemTableContainer.tsx:187]: No clue why `TS` was yelling about insufficient types from `Tooltip`. It looked like `t()` is undefined on initialization hence the errors but that shouldn't be true to how we load the translations 🤔

Comment (viktorgullmark) [reply ExilenceNextApp/src/components/item-table/ItemTableContainer.tsx:187]: Strange indeed, maybe some types got updated, either way this is fine I guess, we can revisit this later to see if we can clean it up!

File: ExilenceNextApp/src/index.tsx (hunk 4 above)
Comment (viktorgullmark) [reply ExilenceNextApp/src/index.tsx:52]: Oof, interesting..

File: ExilenceNextApp/src/utils/snapshot.utils.ts (hunk 5 above)
Comment (viktorgullmark) [reply ExilenceNextApp/src/utils/snapshot.utils.ts:55]: Gotcha, thanks for clearing it up!

File: ExilenceNextApp/tsconfig.json (hunk 7 above)
Comment (viktorgullmark) [reply ExilenceNextApp/tsconfig.json:23]: Thats fine, I didnt bother too much with the types in `TableWrapper` as I was pretty eager to get it finished

**Pull request merged**